lib/workers/repository/updates/generate.ts

```diff
@@ -73,13 +73,17 @@ export function generateBranchConfig(
   const depNames: string[] = [];
   const newValue: string[] = [];
   const toVersions: string[] = [];
+  const toValues: Set<string> = new Set();
   branchUpgrades.forEach((upg) => {
     if (!depNames.includes(upg.depName)) {
       depNames.push(upg.depName);
     }
     if (!toVersions.includes(upg.newVersion)) {
       toVersions.push(upg.newVersion);
     }
+    if (!toValues.has(upg.newValue)) {
+      toValues.add(upg.newValue);
+    }
```

Comment:
```suggestion
    toValues.add(upg.newValue);
```
Set will handle duplicates faster. 🤗

Reply:
as I thought ;) thx